src/components/Onboarding/Steps/StepsItem.js

```diff
@@ -31,17 +31,18 @@ function ConfigureStepsItem({ stepNumber, step, label, currentStep }) {
       css={`
         display: flex;
         align-items: center;
-        height: ${5 * GU}px;
+        height: ${type === 'step' ? 5 * GU : 2 * GU}px;
```

Comment:
since we only have two kinds and we are re writing the `type === 'step'` check, we can have a variable outside called `const isStepType = type === 'step'` and then just do  `height: ${isStepType? 5 * GU : 2 * GU}px;`